docs/_includes/engage.html

```diff
@@ -94,30 +94,30 @@ <h4 class="section-subheading">{{ site.data.sitetext.engage.subtext }}</h4>
               </div>
             </div>  
           </div>
-          <!-- Meeting Minutes -->
+          <!-- CNCF Community -->
           <div class="col-lg-4">
             <div class="card-web card hover-up hover-line"> 
               <div class="card-body">
-                <a href="https://bit.ly/3B7uIy9"><h4 class="card-title"><i class="far fa-file cmu"></i> Meeting Minutes</h4>
+                <a href="https://community.cncf.io/service-mesh-performance-community/"><h4 class="card-title"><i class="far fa-comments cmu"></i> Cloud Native Community Group</h4>
                 <p class="card-text">
-                  Have a topic to discuss? Add it to the meeting minutes.
+                  Join the Service Mesh Performance Cloud Native Community Group!
```

Comment:
```suggestion
                  Join the SMP Cloud Native Community Group!
```

Reply:
@soham4abc if you could just do this change then we are good to go.

Reply:
Yeah sure ! Doing it right away